web-local/src/lib/components/workspace/source/SourceWorkspaceHeader.svelte

```diff
@@ -100,16 +103,29 @@
               )}
             </div>
           {/if}
-          <Tooltip location="bottom" distance={8}>
-            <IconButton
-              on:click={() => onRefreshClick(currentSource.tableName)}
-            >
-              <RefreshIcon size="16px" />
-            </IconButton>
-            <TooltipContent slot="tooltip-content">
-              Refresh the source data
-            </TooltipContent>
-          </Tooltip>
+          {#if connector === "file"}
+            <Tooltip location="bottom" distance={8}>
+              <IconButton
```

Comment:
because of the type's descender height, this looks off to me even though it's technically centered. Can you wrap this `IconButton` in a `div` with a `style="transformY(-1px)"`?